eventdata/README.md

```diff
@@ -55,4 +55,4 @@ Copyright 2015-2018 Elasticsearch https://www.elastic.co
 Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at
 
 http://www.apache.org/licenses/LICENSE-2.0
-Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
```

Comment:
Did this change at all? Not sure why git thinks it has changes.

Reply:
I think this is only a whitespace change, i.e. my editor added a previously missing line ending.